### DISCUSSION TITLE
field persontype api orders wc

### DISCUSSION THREAD

**Discussion by gabrielleal7** [woocommerce/woocommerce#38638] [WooCommerce HPOS] [answered]
field persontype in api orders was deprecated ? 
i dont find him in orders api v3

**Comment by vedanshujain** [comment]
I am not sure what persontype is, can link to a documentation or some code usage from an earlier API version?

**Reply by gabrielleal7** [reply]
![image](https://github.com/woocommerce/woocommerce/assets/81471322/2e0d6f3e-ed67-49b9-8cb7-05ae83f9192e)

are these fields, persontype and cpf, is it a plugin by any chance?
In another store, this data does not appear and I cannot identify why.
These fields are widely used in Brazil to issue invoices.

**Reply by vedanshujain** [reply, marked as answer]
Seems like this is coming from a plugin, I am pretty sure it's not part of WooCommerce core. Could it be this plugin: https://br.wordpress.org/plugins/woocommerce-extra-checkout-fields-for-brazil/?

**Reply by gabrielleal7** [reply]
thanks, that was exactly what I needed.

**Reply by gabrielleal7** [reply]
I have another problem, I have two integrations on sites with woocommerce, which are not allowing me to change the status of the order via PUT according to the documentation, but I have done countless others and never had this problem, can you tell me if it is also at the plugin level that allows put requests or some other site configuration?
